torch_xla/core/dynamo_bridge.py

```diff
@@ -377,7 +419,7 @@ def extract_internal(xla_model: torch.fx.GraphModule):
   skip_checking_input_sharding_threashold = xu.getenv_as(
       'XLA_DYNAMO_INPUT_SHARDING_CHECK_THRESHOLD', int, 5)
 
-  def optimized_mod(*args):
+  def optimized_mod(*input_args):
```

Comment:
`optimized_mod` is invoked with every inference call for dynamo path. Can you measure before, and after performance overhead on this diff? Wrapping the entire function around `time.perf_counter()` (basically less exhaustive https://github.com/pytorch/xla/pull/6598) would be sufficient.

Reply:
Sure, is there a specific workload you wanted to measure on?

Reply:
Just anything basic, can be resnet50 for example, but the point is to grasp the overhead.

Reply:
Test PR here to double check my work - https://github.com/changm/pytorch-xla/pull/3

Resnet 50 model in a loop for 100x, run 3x (note, everything done on CPU in a Cloud VM sooo probably noisy). All times in nanoseconds:

With Patch:
254414249.59
256134671.03
254181391.66

Without
255055284.05
255336177.86
256976810.52

Doesn't seem to be much overhead.

Reply:
Hmm, looks like just noise. I'll add some micro-benchmark which just measures overhead to the repo so we will have it standarised. Thanks!